lua/Dockerfile

```diff
@@ -10,10 +10,12 @@ apt-get install -y --no-install-recommends \
   lua5.4=5.4.4-1 \
   liblua5.4-dev=5.4.4-1 \
   luarocks=3.8.0+dfsg1-1
-rm -rf /var/lib/apt/lists/*
+
+  rm -rf /var/lib/apt/lists/*
```

Comment:
Wrong ident?

Reply:
this commit was just to trigger the CI

not related to the PR itself